format/spec.md

```diff
@@ -483,6 +485,7 @@ Notes:
 2. For `float` and `double`, the value `-0.0` must precede `+0.0`, as in the IEEE 754 `totalOrder` predicate. NaNs are not permitted as lower or upper bounds.
 3. If sort order ID is missing or unknown, then the order is assumed to be unsorted. Only data files and equality delete files should be written with a non-null order id. [Position deletes](#position-delete-files) are required to be sorted by file and position, not a table order, and should set sort order id to null. Readers must ignore sort order id for position delete files.
 4. The following field ids are reserved on `data_file`: 141.
+5. For `geometry`, this a point composed of the min (lower_bound) or max (upper_bound) value of each dimension among all component points of all geometry objects for the file. These can be used for basic pruning only on geometry columns with planar edges. See Appendix D for encoding.
```

Comment:
I think we need to be a little more specific here. The way I read this is that you can take min and max values for each dimension in the point, but that isn't sufficient for spherical edges.

I think this needs to state that the lower and upper bounds must be less than or equal (or greater than or equal) to the values of any point that is located on an edge of the geometry object. In other words, the bounding box must contain all points that are in the geometry object.

If we don't have that requirement, then there could be a point that is outside of the bounding box. If that's the case, then a query that includes the point may not overlap the bounding box and we cannot use it for filtering.

Reply:
Per a previous conversation , it could be beneficial to have it in this form even for spherical edges.  An engine could do some conversion from the bounding box to be useful for spherical edge.  

Do you mean, you do not want this option at all ? (I suppose due to risk of mis-interpreting it)